### PR TITLE
feat: bump bevy to 0.13

### PR DESCRIPTION
**Pull request by liquidhelium**
Maybe I should update the version number?

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -41,7 +41,7 @@ wasm-bindgen-test = { version = "0.3" }
 
 [features]
 all = ["bincode", "ini", "json", "pretty", "ron", "toml", "yaml"]
-default = []
+default = ["all"]
```
Comment (jostly) [inline Cargo.toml:44]: Should probably keep the default features to [] to avoid surprises when most users probably just want to update to the latest bevy.

Comment (umut-sahin) [reply Cargo.toml:44]: I agree. `all` feature is not active by default to reduce binary size and compile times by reducing the number of dependencies. It's meant to be used for prototyping or getting started for a quick demo, it's not for production.

**Comment by umut-sahin** [comment]
Thanks for the PR, I was planning to do it myself this week, and it's a nice surprise!

Could you please revert adding `all` feature to default features and update the commit message to:
```
feat: bump bevy to v0.13
```

I'll start the CI and merge the PR once CI is green afterwards. Thanks!

**Comment by umut-sahin** [comment]
You don't need to worry about the version, I'll bump it before release. Lastly, I'd love to add you as a contributor, if you provide me your `NAME <EMAIL>` like in:
```
authors = [
    "Umut Şahin <umutsahin@protonmail.com>",
    "Shane Lillie <ignignokterr@gmail.com>",
    "Eri Pazos <eri@inventati.org>",
    "Scott Godwin <sgodwincs@gmail.com>",
]
```

File: Cargo.toml (hunk 1 above)
Comment (liquidhelium) [reply Cargo.toml:44]: Oh my fault, I added this for test.

**Comment by liquidhelium** [comment]
I prefer to keep my real name as a secret, so I'm not sure if my pseudonym can be used here.
```
"Helium <liqhelium@gmail.com>"
```

**Comment by liquidhelium** [comment]
Awesome, done!

**Comment by umut-sahin** [comment]
v0.5.0 released https://crates.io/crates/bevy-persistent/0.5.0 with Bevy v0.13 support :partying_face:

**Pull request merged**